configs/sst_display_productivity-flatpak.yaml

```diff
@@ -9,6 +9,8 @@ data:
     - flatpak
     - flatpak-libs
     - flatpak-builder
+    - flatpak-rpm-macros
+    - flatpak-runtime-config
```

Comment:
buildroot only?  If so, maybe move these to the end of the list and precede with a comment?

Reply:
No, they're shipped in CRB.